db/queries.py

```diff
@@ -248,6 +248,7 @@ def insert_fund_data(fund_config):
                 owner_organisation_name=bindparam("owner_organisation_name"),
                 owner_organisation_shortname=bindparam("owner_organisation_shortname"),
                 owner_organisation_logo_uri=bindparam("owner_organisation_logo_uri"),
+                funding_type=bindparam("funding_type"),
```

Comment:
When we are trying to get the data is it sending the finding type as a response ?

Reply:
Yes that's covered by setting the property as an enum in the mock and then [asserting its serialised appropriately here](https://github.com/communitiesuk/funding-service-design-fund-store/pull/342/files#diff-b15f4439c39906d325c2ea3fc3fcd888d521dc825304bcfcf0df3c483763892fR39) (the integration tests could cover a lot more ground by avoiding the mock and just going to the database which is already setup, I might make a follow up to have more confidence in these!)